### DISCUSSION TITLE
Microsoft will stop support for WSA on Windows

### DISCUSSION THREAD

**Discussion by icefrog119** [MustardChef/WSABuilds#331] [General]
so  Microsoft will end support for WSA on windows 11 in 2025, Amazon Appstore already gone starting today, what's going to happen to the development of WSA with Play Store on github?

**Comment by sumoneelse** [comment]
I would appreciate any comments that could point me toward some other solution where I could directly pin & launch an android app from inside windows.  Specifically I would like to avoid having to launch an android environment and *then* launch an Android app (like Mumu and Bluestacks, for example.)

**Reply by MikaelaLovett** [reply]
Me too, please, I use WSA a lot, every day :((

**Reply by MustardChef** [reply]
There no emulator with the feature that you are looking for, for Windows at least.

**Reply by arkni8** [reply]
Yep, I have a couple of messaging app that boot up along with windows and so I mostly get all the notifications. I am not sure any other emulator does that natively. This is flipping bs man. Why the sudden switcheroo. WSA was the only reason I switched to W11.

**Reply by RFLEpicGamer** [reply]
same here, too bad it's getting discontinued soon

**Comment by methanoid** [comment]
I suspect all previous users of WSA will be looking for alternatives. I hope somehow WSA/PlayStore from here can be somehow made to work long term?

**Reply by MustardChef** [reply]
Although WSA will not be updated, I plan to keep providing LTS for WSA 2311.40000.5.0 by keeping the Magisk version, KernelSU version and GApps up to date.

**Reply by methanoid** [reply]
Sorry if this is obvious/dumb but does that mean we can still run WSA after MS switch it off?

**Reply by MustardChef** [reply]
Yes, assuming that Microsoft do not introduce changes in the future, that break WSA.

**Reply by icefrog119** [reply]
hope you can keep the android version updated too, android 14 out, but I believe WSA still running 13

**Reply by acidhorse1972** [reply]
Mine mysteriously has been made defunct and the WSASettings app was replaced with @{MicrosoftCorporationII.WindowsSubsystemForAndroid_2305.40000.6.0_x64__8wekyb3d8bbwe?ms-resource://MicrosoftCorporationII.WindowsSubsystemForAndroid/Resources/WsaDisplayName} in the "Apps > Installed apps" section  of Windows 11 settings.
I hadn't gotten around to upgrading the version number but it's weird like Microsoft sabotaged me. But its the same behavior that I have seen with the VMware Workstation 17, they must really hate VMware.

**Reply by AlexDev404** [reply]
My solution was to think of ways of implementing a sort of custom ROM since the subsystem is already there in-place.

**Comment by Philaupatte** [comment]
Hi,
Well, I am desperate too and hope this project will remain live as long as possible. Please forgive me if this is not the right place to ask but 
nothing ventured, nothing gained. 
I really need bluetooth working within WSA. For sure it will not come via Microsoft. 
Looking at the future of WSA, I suppose it will never come out?
Thanks

**Reply by giftigdegen** [reply]
Is bluetooth not supported? One of the major driving forces that got me to finally upgrade to Windows 11 was that I could never get WSA on Win 10 to work without some pretty big bugs. All of those are fixed in Win 11. But I didn't know BT wasn't supported, and the only apps I _need_ access to via WSA require bluetooth. One app in particular was literally the driving force behind this upgrade.
